frontend/src/metabase/visualizations/click-actions/Mode/Mode.ts

```diff
@@ -34,21 +41,30 @@ export class Mode {
     const question = this._question;
     const props = { question, settings, clicked, extraData };
 
-    const actions = [
+    let actions = [
       ...(mode.hasDrills
         ? queryDrill(question, clicked, this.isDrillEnabled)
         : []),
       ...(mode.clickActions?.flatMap(drill => drill(props)) ?? []),
     ];
 
     if (!actions.length && mode.fallback) {
-      return mode.fallback(props);
-    } else {
-      return actions;
+      actions = mode.fallback(props);
     }
+
+    if (this._extensions?.mapClickActions) {
+      actions = this._extensions.mapClickActions(actions, {
+        value: clicked.value,
+        column: clicked.column,
+        event: clicked.event,
+        data: clicked.data,
```

Comment:
Could we add some doc comments on what these parameter (value, column, event, data) is for? Those field names sounds quite generic, and I couldn't see any doc comments in the `ClickObject` type that would indicate to the SDK user on how they can map them.